packages/components/src/components/post-linkarea/post-linkarea.tsx

```diff
@@ -1,30 +1,49 @@
 import { Component, Element, h, Host } from '@stencil/core';
 import { version } from '@root/package.json';
 
+const INTERACTIVE_ELEMENTS = ['a'].join(',');
```

Comment:
I added this constant so that we can refactor the component in the future to also work with buttons (this will be needed for interactive cards). But for now it only accepts anchor elements as it did before.